pkg/framework/plugins/removepodsviolatingtopologyspreadconstraint/topologyspreadconstraint.go

```diff
@@ -150,6 +150,12 @@ func (d *RemovePodsViolatingTopologySpreadConstraint) Balance(ctx context.Contex
 		// ...where there is a topology constraint
 		var namespaceTopologySpreadConstraints []topologySpreadConstraint
 		for _, pod := range namespacedPods[namespace] {
+
+			// Ignore Completed or Failed Pods.
+			if pod.Status.Phase == v1.PodSucceeded || pod.Status.Phase == v1.PodFailed {
+				continue
+			}
+
```

Comment:
thoughts on moving this logic to line 90? The podFilter used when calling `podutil.ListPodsOnNodes(nodes, d.handle.GetPodsAssignedToNodeFunc(), d.podFilter)`

```
podFilter = podutil.WrapFilterFuncs(podFilter, func(pod *v1.Pod) bool {
  return pod.Status.Phase != v1.PodSucceeded && pod.Status.Phase != v1.PodFailed
})
```

Reply:
@h4wkmoon what do you think? Let me know if I can help with the unit tests